packages/css/dropdownmenu.css

```diff
@@ -14,30 +10,25 @@
   z-index: 1500;
   margin: 0;
   list-style: none;
-  border-radius: var(--dsc-dropdownmenu-border-radius);
-  box-shadow: var(--dsc-dropdownmenu-shadow);
-  background-color: var(--dsc-dropdownmenu-background);
-  border: var(--dsc-dropdownmenu-border);
+  border-radius: min(1rem, var(--ds-border-radius-md));
+  box-shadow: var(--ds-shadow-md);
+  background-color: var(--ds-color-neutral-background-default);
+  border: 1px solid var(--ds-color-neutral-border-subtle);
   min-width: var(--dsc-dropdownmenu-min-width);
-}
-
-.ds-dropdownmenu--sm {
-  --dsc-dropdownmenu-padding: var(--ds-spacing-2);
-  --dsc-dropdownmenu-min-width: 240px;
-}
 
-.ds-dropdownmenu--md {
-  --dsc-dropdownmenu-padding: var(--ds-spacing-3) var(--ds-spacing-2);
-  --dsc-dropdownmenu-min-width: 260px;
-}
+  &[data-size='sm'] {
+    --dsc-dropdownmenu-padding: var(--ds-spacing-2);
+    --dsc-dropdownmenu-min-width: 240px;
```

Comment:
Can this be `rem` so we meet accessibility requirements for text zoom? (also on `data-size="lg"`)

Reply:
@Thunear Is `15rem` fine here?

Reply:
Yes, that should be fine.

Reply:
I did some rounding, and we now have

| size| old| new|
|--------|--------|--------|
| sm| 240px | 15rem |
| md | 260px | 16rem |
| lg | 280px | 18rem |